modules/setup_timer/CfgNotifications.hpp

```diff
@@ -1,9 +1,9 @@
-class SetupTimerEnded {
-	title = "SETUP TIMER";
-	iconPicture = "\a3\Ui_f\data\GUI\Cfg\Notifications\tridentEnemy_ca.paa";
-	description = "<t>Setup Timer Over, Fight!</t>";
-	color[] = {1,1,1,1};
-	colorIconPicture[] = {0.5,0,0,0.5};
-	colorIconText[] = {1,1,1,1};
-	duration = 5;
+class GVAR(TimerEnded) {
+    Title = "SETUP TIMER";
```

Comment:
Does it matter that this field starts with upper case and the rest are lower case?

Reply:
no, but pretty